api/envoy/config/core/v4alpha/protocol.proto

```diff
@@ -172,7 +172,7 @@ message Http2ProtocolOptions {
 
     // The 16 bit parameter identifier.
     google.protobuf.UInt32Value identifier = 1 [
-      (validate.rules).uint32 = {lte: 65536 gte: 1},
+      (validate.rules).uint32 = {lte: 65535 gte: 0},
```

Comment:
Why was the lower bound decreased?

Reply:
Because the datatype is  actually uint16 which ranges from 0-65535. And in the protocol I didn't see any restriction saying it should be gte 1. 
https://tools.ietf.org/html/rfc7540#section-6.5.1

Reply:
The only settings identifiers that exist are 0x1-0x6, but the spec doesn't explicitly guard against a 0 identifier, or those above 6.

The use of this is solely for configuration, so it won't impact anyone who is actually using for valid SETTINGs.

Reply:
Will defer this to API review.